### PR TITLE
[FPADF] Announcement

### PR DESCRIPTION
**Pull request by WandererFan**
An announcement of FPADF and description of process.  re issue #24

### CODE REVIEW

File: FPADF_Announcement.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## <img src="images/freecad.svg" style="zoom:50%;" /> FreeCAD Project Association Development Fund
+
+
+#### Announcement - FPA Development Fund
+
+The FreeCAD Project Association is pleased to announce the creation of the FPA Development Fund (FPADF). The FPADF offers grants to individuals for contributions to FreeCAD.
+
+Initially, grants from the FPADF will be capped at US$1000 and grants will be awarded only for work on the FreeCAD code base.
```
Comment (sliptonic) [inline FPADF_Announcement.md:8]:
```suggestion
Grants are capped at US$1000 and will be awarded only for work on the FreeCAD code base.
```

Comment (sliptonic) [reply FPADF_Announcement.md:8]: Avoid casting the current announcment as suboptimal or inferior to a future goal.

File: FPADF_Announcement.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+The cap may be raised as the FPA gains experience in the awarding of grants and as available funds allow.
+
+It is expected that grants will eventually be made available to contributors working on documentation, packaging, and other aspects of FreeCAD.
```
Comment (sliptonic) [inline FPADF_Announcement.md:12]:
```suggestion

```

File: FPADF_Announcement.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Any contributor who wishes to be considered for a grant must submit a proposal in writing to the FPA. The contributor must state clearly how much money is requested and what the contributor intends to create/improve/fix. An estimated timeline with major milestones should be part of the grant application and clear completion criteria should be provided.
```
Comment (sliptonic) [inline FPADF_Announcement.md:14]:
```suggestion
Any contributor who wishes to be considered for a grant should submit a proposal in writing to the FPA. The proposal should state clearly how much money is requested and what the contributor intends to create/improve/fix. An estimated timeline with major milestones should be included along with clear completion criteria.
```

File: FPADF_Announcement.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+Proposals will be considered by the FPA general body and approved by vote.
+
+Grant applications may be forwarded to any member of the FPA for consideration.
+
```
Comment (sliptonic) [inline FPADF_Announcement.md:19]:
```suggestion
The FreeCAD Project Association would like to expand the number and type of grants given.  Donations made to the FPA and designated DEVELOPMENT FUND will be used for this purpose. 
```

**Review by sliptonic — CHANGES_REQUESTED** [review]
Great start!  This needs to happen.

**Comment by chennes** [comment]
Do we say someplace else how exactly one "submits a proposal to the FPA"?

**Comment by WandererFan** [comment]
> Do we say someplace else how exactly one "submits a proposal to the FPA"?

Just the bit about "forward to any member of the FPA".   We don't have an FPA email?  We could add "via private message on the FreeCAD forum".

**Review by sliptonic — APPROVED** [review]

**Comment by sliptonic** [comment]
> What next? Merge this PR. Post in FC Forum. Post in blog.
> 
> Do we need wider approval first?

Let's roll with it.  It looks good to me and has had sufficient time for feedback.  It makes a great blog post.  A forum post and twitter post linking to the blog post would be great.

**Pull request merged**